operations/bills.md

```diff
@@ -104,13 +109,15 @@ Returns all bills, possibly filtered by customers, identifiers and other filters
                 }
             }
         }
-    ]
+    ],
+    "Cursor": "8d02142f-31cf-4115-90bf-ae5200c7a1ba"
 }
 ```
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `Bills` | array of [Bill](#bill) | required | The filtered bills. |
+| `Cursor` | string | required | Unique identifier of the last and hence oldest bill returned. This can be used in [Limitation](messagethreads.md#limitation) in a subsequent request to fetch the next batch of older bills. |
```

Comment:
Link to the correct page `../guidelines/pagination.md#limitation` instead of `messagethreads.md`